Makefile

```diff
@@ -141,14 +141,10 @@ package: build-if-changed
 
 .PHONY: package-win32
 package-win32:
-	@echo Packaging .exe...
+	@echo "Packaging exe..."
 	@npx electron-builder --win -p $(PUBLISH)
-ifeq ($(IS_WINDOWS),true)
```

Comment:
I decided to remove the platform check under the assumption these will mostly run in CI.

I'm not aware of any requirement for building Windows on a non Windows machine either.

This way, the script is simpler to understand.